dspace-api/src/main/java/org/dspace/xmlworkflow/state/actions/userassignment/ClaimAction.java

```diff
@@ -115,7 +115,10 @@ public void regenerateTasks(Context c, XmlWorkflowItem wfi, RoleMembers roleMemb
         } else {
             log.info(LogHelper.getHeader(c, "warning while activating claim action",
                                           "No group or person was found for the following roleid: " + getParent()
-                                              .getStep().getId()));
+                                              .getStep().getId()) + ". The item will be sent back to submitter");
+
+            XmlWorkflowServiceFactory.getInstance().getXmlWorkflowService().sendWorkflowItemBackSubmission(
+                    c, wfi, c.getCurrentUser(), this.getProvenanceStartId(), "");
```

Comment:
not sure that this is appropriate. If a group is empty the workflow step should be skipped

Reply:
> workflow step should be skipped

I will see to it. I failed to comprehend what exactly is the whole process so I opted for this since it is still the suggested fix.